### PR TITLE
update Readme

### PR DESCRIPTION
**Pull request by H3arn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,130 @@
...
+
+  As for the acquisition of "chat_id", you can search Google.
+
+
+
+This usage will be mentioned below.
+
+## Deploy
+
+Type `github-webhook-notification_linux_amd64 --help` to get more usages.
```
Comment (KunoiSayami) [inline README.md]:
```suggestion
Type `github-webhook-notification --help` to get more usages.
```

Should be changed to platform irrelevant

File: README.md [hunk 2]
```diff
@@ -1,6 +1,130 @@
...
+
+It's OK to simply run it. For example:
+
+```sh
+github-webhook-notification_linux_amd64 -c data/config.toml
```
Comment (KunoiSayami) [inline README.md]:
```suggestion
github-webhook-notification -c data/config.toml
```

File: README.md [hunk 3]
```diff
@@ -1,6 +1,130 @@
...
+
+  is optional. 
+
+  Should correspond to the "secret" field value in your GitHub webhook settings.
+
+- `bot_token` 
+
+  is the bot token of your Telegram bot. 
+
+  You can find it in  Telegram@Botfather.
```
Comment (KunoiSayami) [inline README.md]:
```suggestion
  You can find it in  [Telegram@Botfather](https://t.me/botfather). 
```

File: README.md [hunk 4]
```diff
@@ -1,6 +1,130 @@
...
+# ./data/config.toml
+# /usr/local/etc/gh-wbhk-tg/config.toml
+[server]
+bind = ""
+port = 11451
+secrets = "1145141919810"
+
+[telegram]
+bot_token = "1145141919:810abcdefg"
+send_to = [114514, "1919810"]
```
Comment (KunoiSayami) [inline README.md]:
```suggestion
send_to = [114514, 1919810]
```
The quotes are just for testing

File: README.md [hunk 5]
```diff
@@ -1,6 +1,130 @@
...
+git clone https://github.com/KunoiSayami/github-webhook-notification.rs.git
+cd github-webhook-notification.rs
+cargo build --release
+```
+
+Then go to `./target` and you will find the executable binary file. Copy it to the place you want to destinate it to.
+
+
+
+## Install From Pre-built Executable
```
Comment (KunoiSayami) [inline README.md:25]: This chapter is actually not very necessary

**Review by KunoiSayami — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (H3arn) [reply README.md:25]: But you know that some instances are not qualified for compiling the source code.
Sometimes they do need a prebuilt release.

Comment (KunoiSayami) [reply README.md:25]: You can just place a link here, they should know what to do.

Comment (KunoiSayami) [reply README.md:25]: like `You can get pre-build executable here: balabala`

Comment (H3arn) [reply README.md:25]: OK I got this.

**Pull request merged**